activerecord/lib/active_record/relation/batches.rb

```diff
@@ -251,20 +254,37 @@ def in_batches(of: 1000, start: nil, finish: nil, load: false, error_on_ignore:
           end
         end
 
-        batch_relation = relation.where(arel_attribute(primary_key).gt(primary_key_offset))
+        where_condition =
+        batch_relation = relation.where(
+          order == :desc ?
```

Comment:
This won't work if order is `"desc"`, `:DESC`, etc.

We should either document and validate that only `:asc` or `:desc` may be passed, or make the parameter a boolean, e.g. `reverse_order: true`.

Reply:
@eugeneius thanks, will change parameter.